api/api.go

```diff
@@ -68,7 +68,7 @@ func NewAPI(store *event.Store, drivers map[string]driver.Driver, port int) *API
 
 	// retrieve overall status
 	mux.Handle(fmt.Sprintf("/%s/%s", defaultAPIVersion, overallStatusPath),
-		newOverallStatusHandler(store, defaultAPIVersion))
+		newOverallStatusHandler(store, drivers, defaultAPIVersion))
```

Comment:
4!

Reply:
:nod-intensifies:

Reply:
???

Reply:
Sorry for the missing context! This is the 4th place where I've passed in 'drivers' which is a `map[string]driver` and has the danger of being modified. We want to refactor the map into something an actual `Drivers{}` object. I'm currently planning to do this after Beta to respect the code freeze and testing. Open to feedback on the timing!

Reply:
This is in context `map[string]drivers` across the recent series of PRs

https://github.com/hashicorp/consul-terraform-sync/pull/190#discussion_r578003640
https://github.com/hashicorp/consul-terraform-sync/pull/202#discussion_r580399453

Reply:
Thanks!